packages/www/pages/app/admin/keys.tsx

```diff
@@ -10,7 +10,7 @@ import { getTabs } from "../admin";
 const Keys = () => {
   useLoggedIn();
   const { user, logout } = useApi();
-  if (!user || user.emailValid === false) {
+  if (!user) {
```

Comment:
Example of the parameter I was discussing:

```suggestion
  if (!user || user.emailValid === req.config.requireEmailVerification) {
```

Reply:
Maybe we need to check both separately? Not sure what we want to happen on a `emailValid = true` and `req.config.requireEmailVerification = false` case.

```suggestion
  if (!user || !user.emailValid || !req.config.requireEmailVerification) {
```